namespaces/http.yml

```diff
@@ -0,0 +1,59 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: http
```

Comment:
Then is this some sort of "openstack eventlet/wsgi service log format" or something like that?